axis_backend/apps/clients/models/activity.py

```
@@ -0,0 +1,107 @@
+        db_table = 'client_activities'
+        verbose_name = 'Client Activity'
+        verbose_name_plural = 'Client Activities'
+        ordering = ['-activity_date', '-created_at']
+        indexes = [
+            models.Index(fields=['client', '-activity_date']),
+            models.Index(fields=['client', 'activity_type']),
+            models.Index(fields=['activity_type', '-activity_date']),
+            # models.Index(fields=['staff_member', '-activity_date']),  # TODO: Uncomment when staff app exists
+        ]
+
+    def __str__(self):
```

Ruff (0.14.6)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)